### PR TITLE
contacts automation test cases

### PR DESCRIPTION
**Pull request by stairaku**

### CODE REVIEW

File: testing/katalon/Object Repository/Contacts/createIndividualContact/addAnotherEmailAddressLink.rs [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jdtoombs) [inline testing/katalon/Object Repository/Contacts/createIndividualContact/addAnotherEmailAddressLink.rs:1]: @stairaku 
Not sure if it matters but I can show you how to substantially shorten the size of these selectors. I can book some time with you this upcoming sprint to show you

**Comment by devinleighsmith** [comment]
@stairaku don't forget to assign yourself and add labels to the PR!

File: testing/katalon/Object Repository/Contacts/createIndividualContact/addAnotherEmailAddressLink.rs (hunk 1 above)
Comment (stairaku) [reply testing/katalon/Object Repository/Contacts/createIndividualContact/addAnotherEmailAddressLink.rs:1]: for sure! and also I don't know how to assign these PRs to myself and add labels. That would also be helpful. Thanks!

Comment (devinleighsmith) [reply testing/katalon/Object Repository/Contacts/createIndividualContact/addAnotherEmailAddressLink.rs:1]: Has this been addressed yet?

Comment (stairaku) [reply testing/katalon/Object Repository/Contacts/createIndividualContact/addAnotherEmailAddressLink.rs:1]: Not yet. I'm coordinating with Jonny to check this out

Comment (devinleighsmith) [reply testing/katalon/Object Repository/Contacts/createIndividualContact/addAnotherEmailAddressLink.rs:1]: 21 days now! can we merge this?

Comment (stairaku) [reply testing/katalon/Object Repository/Contacts/createIndividualContact/addAnotherEmailAddressLink.rs:1]: yes please! Sorry about the delay

**Review by jdtoombs — APPROVED** [review]

**Pull request closed without merging**